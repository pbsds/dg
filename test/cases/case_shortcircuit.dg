print 1
  (print 2) or  (print 3) or  (print 4)
print 5
  (print 6) =>  (print 7) =>  (print 8)
print 9
  (print 10) and (print 11) and (print 12)

print!

print 13
  (print 14 or 15) or  (print 16 or 17) or  (print 18 or 19)
print 20
  (print 21 or 22) =>  (print 23 or 24) =>  (print 25 or 26)
print 27
  (print 28 or 29) and (print 30 or 31) and (print 32 or 33)
