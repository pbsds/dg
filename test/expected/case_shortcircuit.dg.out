2
3
4
1 None
6
5 None
10
9 None

14
13 15
21
23
25
20 26
28
30
32
27 33
